### PR TITLE
run gofmt; pass golint

### PR DESCRIPTION
**Pull request by pd**

### CODE REVIEW

File: pgmgr/config_test.go [hunk 1]
```diff
@@ -76,14 +78,14 @@ func TestOverlays(t *testing.T) {
...
 
 	if c.Port != 123 {
 		t.Fatal("config's port should not change, but was", c.Port)
 	}
 }
 
-func TestURL (t *testing.T) {
+func TestURL(t *testing.T) {
 	c := &pgmgr.Config{}
 	c.Url = "postgres://foo@bar:5431/testdb"
```
Comment (rnubel) [inline pgmgr/config_test.go]: `s/Url/URL/g`, to fix the tests.

**Comment by rnubel** [comment]
:heart:

**Comment by pd** [comment]
Url->URL fixed, i actually cherry picked this out of a different branch and just missed it

**Comment by rnubel** [comment]
Sweet, thanks.

**Pull request merged**